app/src/main/java/org/oppia/android/app/privacypolicytermsofservice/PrivacyPolicyActivity.kt

```diff
@@ -0,0 +1,31 @@
+package org.oppia.android.app.privacypolicytermsofservice
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.ActivityComponentImpl
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The Privacy Policy page activity for placement of single PRIVACY_POLICY. */
```

Comment:
```suggestion
/** The activity for displaying the app's privacy policy. */
```

Ditto (i.e. wording suggestions) for the terms of service activity.

Reply:
done.